test/caching.bats

```diff
@@ -185,6 +185,7 @@ EOF
 }
 
 @test "can read previous version's cache" {
+    skip "old stacker will not build because libsquashfs1-dev is not installed"
```

Comment:
No, we don't want to disable this in general right?
We expect this test alone to fail and we can force-merge.

Reply:
The intent I believe is PRs don't break backward compatibility. We can disable this test for now, but we need a new PR to re-enable it right afterwards.